gdcdictionary/schemas/_terms.yaml

```diff
@@ -2368,15 +2369,16 @@ loci_count:
 locus:
   common:
     description: >-
-      Alphanumeric value used to describe the locus of a specific genetic variant.
-      Example: NM_001126114.
+      The position of a gene or a chromosomal marker on a chromosome; also, a stretch
+      of DNA at a particular place on a particular chromosome. The use of locus is
+      sometimes restricted to mean regions of DNA that are expressed (example: 17p13.1.).
     termDef:
       term: >-
-        Molecular Laboratory Procedure Locus Location Text
-      source: caDSR
-      cde_id: 6142506
-      cde_version: 1.0
-      term_url: "https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search?publicId=6142506&version=1.0"
+        Locus
+      source: NCIt
+      cde_id: C45822
+      cde_version: 22.04d
+      term_url: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C45822"
```

Comment:
I'm not sure if this has changed since the ticket was created but the term_url specified in DEV-179 is: https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&version=22.04d&ns=ncit&code=C45822&key=1930938629&b=1&n=null

Reply:
i think they both take you to the same place, so either one is fine. i don't have a preference